environment.yml

```diff
@@ -24,6 +24,7 @@ dependencies:
     - numpy[version='>=1.20.0,<=1.23'] # Upper limit needed for Apple silicon
     - numpydoc # Needed for documentation
     - pandas
+    - pastasoss --no-deps
```

Comment:
If this works, this should be added to the `setup.cfg` file

Reply:
Nope.  The test ignored `--no-deps`